auth-web/src/components/auth/mixins/AccountMixin.vue

```diff
@@ -52,5 +52,9 @@ export default class AccountMixin extends Vue {
   protected get isSbcStaffAccount (): boolean {
     return this.currentOrganization?.orgType === Account.SBC_STAFF
   }
+
+  protected get isBasicAccount (): boolean {
```

Comment:
Is this used?

Reply:
it used before I removed account type check, I'll move it.